docs/manual/docs/contributing/doing-a-release.md

```diff
@@ -146,9 +194,9 @@ Once the release branch has been thoroughly tested and is stable a release can b
     cd /home/frs/project/g/ge/geonetwork/GeoNetwork_opensource
     # or for RC release
     cd /home/frs/project/g/ge/geonetwork/GeoNetwork_unstable_development_versions/
-    mkdir v3.12.1
-    cd v3.12.1
-    put docs/changes3.12.1-0.txt
+    mkdir v3.12.12
+    cd v3.12.12
+    put docs/changes3.12.12-0.txt
     put release/target/GeoNetwork*/geonetwork-bundle*.zip*
     put web/target/geonetwork.war*
     bye
```

Comment:
I am personally not sure why ``bye`` does, is it like ``return``

Reply:
Both `bye` and `quit` are valid commands to close the ftp connection.